docs/docs/creating-a-starter.md

```diff
@@ -35,6 +37,10 @@ Although the official starters live in the Gatsby repo, community members can of
 
 `gatsby new my-app https://github.com/netlify-templates/gatsby-starter-netlify-cms`
 
+## Open source license
+
+Gatsby recommends all starters use the [BSD Zero Clause License](https://choosealicense.com/licenses/0bsd/)(0BSD). While the [MIT License](https://choosealicense.com/licenses/mit/) is more common (and was used in Gatsby official starters [for a long time](https://github.com/gatsbyjs/gatsby/pull/25441)), MIT's "License and copyright notice" condition make it a bad fit for starters. Starters, by design, are for people to customize and use for their own sites (open and closed source). The 0BSD has no "License and copyright notice" condition. This allows folks to use and customize the starter without reference to the original license or source. Gatsby believes this is how most creators expect starters to be treated and how developers expect to use starters, despite many having the MIT license. By recommending 0BSD we are trying to align expectations and licensing.
```

Comment:
```suggestion
Gatsby recommends all starters use the [BSD Zero Clause License](https://choosealicense.com/licenses/0bsd/)(0BSD). While the [MIT License](https://choosealicense.com/licenses/mit/) is more common (and was used in Gatsby official starters [for a long time](https://github.com/gatsbyjs/gatsby/pull/25441)), MIT's "License and copyright notice" condition make it a bad fit for starters. Starters, by design, are for people to customize and use for their own sites (open and closed source). The 0BSD has no "License and copyright notice" condition. This allows folks to use and customize the starter without reference to the original license or source. This is how most creators expect starters to be treated and how developers expect to use starters, despite many having the MIT license. Using 0BSD represents better alignment between expectations and licensing.
```

I like to avoid personifying Gatsby whenever possible.